### ISSUE TITLE
Lastest Build failures

### ISSUE THREAD

**Issue by zz85** [openssl/openssl#11487]
The latest merge (#11147) seems to have failing builds on Ubuntu

```sh
+ unzip OpenSSL_1_1_1-stable.zip
+ cd openssl-OpenSSL_1_1_1-stable
+ CONFIGURE='./config -d'
+ ./config -d -g3 -fPIC no-md2 no-rc5 no-rfc3779 no-sctp no-ssl-trace no-zlib no-hw no-mdc2 no-seed no-idea enable-ec_nistp_64_gcc_128 no-camellia no-bf no-ripemd no-dsa no-ssl2 no-ssl3 no-capieng -DSSL_FORBID_ENULL -DOPENSSL_NO_DTLS1 -DOPENSSL_NO_HEARTBEATS --prefix=/codebuild/output/src892132891/src/github.com/awslabs/s2n/test-deps/openssl-1.1.1
***** Deprecated options: no-ripemd
+ make -j 72 depend
+ make -j 72
crypto/rand/drbg_ctr.c: In function 'inc_128':
crypto/rand/drbg_ctr.c:25:5: error: unknown type name 'u32'
     u32 n = 16, c = 1;
     ^~~
crypto/rand/drbg_ctr.c:30:17: error: 'u8' undeclared (first use in this function)
         p[n] = (u8)c;
                 ^~
crypto/rand/drbg_ctr.c:30:17: note: each undeclared identifier is reported only once for each function it appears in
crypto/rand/drbg_ctr.c:30:20: error: expected ';' before 'c'         p[n] = (u8)c;
                    ^
make[1]: *** [crypto/rand/drbg_ctr.o] Error 1
```

**Comment by zz85** [comment]
I can replicate this on a Ubuntu 18.04.2 LTS

**Comment by bernd-edlinger** [comment]
I think this is fixed by 0d011f540400b425aba1c3e59624ad9dbabe83cb
